install.sh

```diff
@@ -255,21 +228,20 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   rest="${tarfile#*-}"    # extract string after first '-'
   version="${rest%%-chromeos*}"
                           # extract string between first '-' and "-chromeos"
-
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
-## workaround https://github.com/chromebrew/chromebrew/issues/3305
+# Work around https://github.com/chromebrew/chromebrew/issues/3305
 sudo ldconfig &> /dev/null || true
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+echo -e "\e[0;37m" # Grey
```

Comment:
What is wrong with using variables?  This way is not elegant.

Reply:
Because the variables are unnecessary for all of the other calls, and creating one variable in contrast to the other integrated calls isn't elegant either.

Reply:
What do you mean?  This color is referenced multiple times so it makes sense to create a variable for it.  Also, the variable abstracts the value so if it ever changes in the future, it only needs to be changed on one line.

Reply:
The refactoring removed all of the color variables, so adding one back wouldn't look great. I've got another refactoring idea, I'll try it in a bit.

Reply:
I've pushed it, what do you think?